scylla/src/network/connection.rs

```diff
@@ -76,6 +77,25 @@ const LOCAL_VERSION: &str = "SELECT schema_version FROM system.local WHERE key='
 const OLD_ORPHAN_COUNT_THRESHOLD: usize = 1024;
 const OLD_AGE_ORPHAN_THRESHOLD: std::time::Duration = std::time::Duration::from_secs(1);
 
+/// Represents a write coalescing delay configuration option.
+#[derive(Clone)]
+#[non_exhaustive]
+pub enum WriteCoalescingDelay {
```

Comment:
:wrench: Let's derive `Debug`, too. It's often the case that we later discover it's needed for, e.g., some `unwrap()`, and we add it anyway.